index.html

```diff
@@ -3,50 +3,199 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
   <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
 </head>
 
 <body>
-  <header class="toolbar">
-    <a href=# id="toolbar-name">Richard Burneo</a>
-    <i id="hamburguer" class="fa fa-bars" aria-hidden="true"></i>
+  <header id="brand" class="toolbar">
+    <div class="my-logo">
+      Richard Burneo
+    </div>
+    <nav class="menu-botton">
+      <i class="fa-solid fa-bars"></i>
+    </nav>
   </header>
   <section class="headline">
-    <h1>Hey There I'm Richard</h1>
-    <h3 id="h3-headline">I am Software Developer</h3>
-    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like
-      what
-      you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <div class="information">
+      <h1 class="my-name">Hey there. <br>I'm Richard</h1>
+      <h2 class="iam">I'm Sotfware Developer</h2>
+      <p class="my-description">I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
     <div class="social-network">
       <ul>
         <li>
-          <a href="https://github.com/pirc25" target="_blank" rel="noopener noreferrer"><i
-              class="fa-brands fa-github social"></i></a>
+          <i class="fa-brands fa-github"></i>
         </li>
         <li>
-          <a href="https://www.linkedin.com/in/richard-burneo-aguilera-1277aa203/" target="_blank"
-            rel="noopener noreferrer"><i class="fab fa-linkedin-in social"></i></a>
+          <i class="fab fa-linkedin-in"></i>
         </li>
         <li>
-          <a><i class="fa-brands fa-angellist social"></i></a>
+          <i class="fa-brands fa-angellist"></i>
         </li>
         <li>
-          <a href="https://twitter.com/RichardBurneo" rel="noopener noreferrer" target="_blank"><i
-              class="fa-brands fa-twitter social"></i></a>
+          <i class="fa-brands fa-twitter"></i>
         </li>
         <li>
-          <a href=#><i class="fa-solid fa-m social"></i></a>
+          <i class="fa-solid fa-m"></i>
         </li>
       </ul>
     </div>
   </section>
 
-
+  <section class="work-section grid">
+    <div class="work-one">
+      <h2 class="title-work-one">My Recent Works</h2>
+      <hr>
+      <img class="image-recent-work" src="./images/Placeholder@2x.png" alt="This is a meaningful image description">
```

Comment:
- [ ] I would implore you to use this image as a background image because decorations are always backgrounded images, not img tags.